src/util/Util.js

```diff
@@ -573,6 +573,34 @@ class Util {
       setTimeout(resolve, ms);
     });
   }
+  
+  /**
+   * Checks whether there is a custom emoji in the message content
+   * @param {string} text to check
+   * @returns {Booelan}
+   */
+  static includesEmoji(text) {
+        if (text !== string) throw new TypeError("Value is not a string")
+        if (text.match(/<?(a)?:?(\w{2,32}):(\d{17,19})>?/)) {
+            return true
+        } else {
+            return false
+        }
+    }
+  
+ /** 
+ * Check whether there is a unicode emoji in the message content
+ * @param {string} text to check
+ * @returns {Boolean}
+ */
+ static includesUnicodeEmoji(text) {
```

Comment:
Even if I thought this was a good method, which I don't, hardcoding this Unicode regex is a bad idea. It's not something this library should have to maintain.

*Theoretically* if implementing this in my own code, I'd propose using an external lib focused on unicode emoji to define this. I don't think discord.js should be doing this at all though as unlike custom emojis, this is not Discord related functionality.